tools/bin/commands/util/openshift_funcs

```diff
@@ -11,7 +11,7 @@ check_oc_version()
     local minimum=${OC_MIN_VERSION}
     local test=$(oc version | grep "^oc" | tr -d oc\ v | cut -f1 -d "+")
     if [ "$test" = "" ]; then
-        local test=$(oc version | grep 'Client Version' | sed "s/^.*GitVersion:\"v\(.*\)\", GitCommit.*$/\1/")
+        local test=$(oc version | grep 'Client Version' | sed "s/^.*GitVersion:\"v\(.*\)\", GitCommit.*$/\1/" | sed "s/Client Version: v//")
```

Comment:
For oc downloaded from https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/latest/:

```
$ ./oc version
Client Version: openshift-clients-4.2.0-201910041700
Kubernetes Version: v1.11.0+d4cacc0
$ ./oc version | grep 'Client Version' | sed "s/^.*GitVersion:\"v\(.*\)\", GitCommit.*$/\1/" | sed "s/Client Version: v//"
Client Version: openshift-clients-4.2.0-201910041700
```

In that case this might work better:

```suggestion
        local test=$(oc version | grep -i 'client version'|sed -e 's/.*: \(openshift-clients-\)\{0,1\}//')
```

Might be a good idea to add tests for this.

Also of note is that there's a yaml/json output in the 4.2 of `oc version`, doesn't seem to be much useful at the moment:

```
$ ./oc version --client -o json
{
  "clientVersion": {
    "major": "",
    "minor": "",
    "gitVersion": "openshift-clients-4.2.0-201910041700",
    "gitCommit": "d5465d715197862b15c8cf05e864d7c3cfea6917",
    "gitTreeState": "clean",
    "buildDate": "2019-10-07T15:16:39Z",
    "goVersion": "go1.12.8",
    "compiler": "gc",
    "platform": "linux/amd64"
  }
}
```

Reply:
The suggested change broke when dealing with crc due to the 'v' so taken it to bits and managed to do a single regexp with grep. Added some bats tests to prove it works.
WDYT?
@zregvart